.github/workflows/ci-build.yml

```diff
@@ -15,18 +15,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Install Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.17.x
+        go-version: 1.18.x
         
     - name: Checkout code
       uses: actions/checkout@v2
     
     - name: Lint
-      uses: golangci/golangci-lint-action@v2
+      uses: golangci/golangci-lint-action@v3
       with:
-        version: v1.43.0
-        skip-go-installation: true # we already installed Go with `actions/setup-go@v2` above
```

Comment:
is there any reason for removing the `skip-go-installation` flag here?

Reply:
seem like since v3 they are not installing go anymore by default [v3 PR](https://github.com/golangci/golangci-lint-action/pull/403)